tests/models/test_triplestoreconfig.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for `oteapi.models.triplestoreconfig`"""
+import pytest
+
+
+def test_triplestoreconfig():
+    """Pytest for TripleStoreConfig."""
+
+    import json
+
+    from oteapi.models.triplestoreconfig import TripleStoreConfig
+    from oteapi.settings import settings
+
+    description = """TripleStore Configuration.
+
+    This is a configuration for the
+    [`TripleStore`][oteapi.triplestore.triplestore.TripleStore].
+
+    This class should not be used directly as a configuration object
+    for a strategy object, but only as a configuration field inside
+    a configuration object.
+    """
+
+    config = dict(
+        agraphHost="localhost",
+        agraphPort=8080,
+        user="abc",
+        password="pass",
+        repositoryName="test",
+    )
```

Comment:
```suggestion
    config = {
        "agraphHost": "localhost",
        "agraphPort": 8080,
        "user": "abc",
        "password": "pass",
        "repositoryName": "test",
    )
```
It's more efficient to use dict comprehensions (`{}`) to define dicts that instantiating through a call to `dict()`.
Although it's not wrong to do the other thing :)

This suggested change goes for all other instances of `dict()` usages as well.